neurobench/examples/mswc_fscil/mswc_fscil_proto.py

```diff
@@ -348,26 +351,23 @@ def pre_train(model):
                                     support_query_split=(100,100),
                                     samples_per_class=200)
 
-
-        few_shot_optimizer = torch.optim.SGD(eval_model.net.parameters(), lr=EVAL_LR, momentum=0.9, weight_decay=0.0005)
-
         # Iteration over incremental sessions
         for session, (support, query, query_classes) in enumerate(few_shot_dataloader):
             print(f"Session: {session+1}")
 
             # Define benchmark object
-            benchmark_all_test = Benchmark(eval_model, metric_list=[static_metrics, data_metrics], dataloader=test_loader, 
+            benchmark_all_test = Benchmark(eval_model, metric_list=[static_metrics, workload_metrics], dataloader=test_loader, 
                                 preprocessors=[to_device, encode, squeeze], postprocessors=[])
 
             benchmark_new_classes = Benchmark(eval_model, metric_list=[[],["classification_accuracy"]], dataloader=test_loader,
                                 preprocessors=[to_device, encode, squeeze], postprocessors=[])
             
-
             cur_class = support[0][1].tolist()
             eval_model.net.cur_j = examples_per_class(cur_class, 200, 5)
```

Comment:
Can be deleted. cur_j is not used

Reply:
About `eval_model.net.cur_j = examples_per_class(cur_class, 200, 5)`

Reply:
Resolved by 37346879901a8255ef2af844f2b42a1e5347c58b